README.md

```diff
@@ -1,11 +1,9 @@
 ## Install
 
 ```bash
-npm install eslint
-npm install eslint-plugin-promise eslint-plugin-standard eslint-plugin-react
-npm install eslint-config-standard
-npm install eslint-config-punkave
+npm install eslint eslint-plugin-promise eslint-plugin-standard eslint-plugin-react eslint-config-standard eslint-config-punkave --save-dev
```

Comment:
How about clarifying that all of this should happen at project level? People are used to installing linters globally.

Also, instructions to get atom to leverage it would be great, if you have them.